crates/ide/src/hover/tests.rs

```diff
@@ -2717,7 +2728,7 @@ fn main() { let s$0t = foo(); }
                                     focus_range: 52..54,
                                     name: "S2",
                                     kind: Struct,
-                                    description: "struct S2",
+                                    description: "struct S2 {\n}",
```

Comment:
These would normally be `struct S2 {}`, but feel free to ignore this if it's too much of a hassle.